custom-dashboards/apply.yml

```diff
@@ -0,0 +1,6 @@
+---
+- hosts: seed-hosts[0]
+  vars:
+    provision: "{{ dashboard_deploy | default(true) }}"
+  roles:
+     - roles/openshift-applier/roles/openshift-applier
```

Comment:
Per the instructions above for `ansible-galaxy` and using `-p galaxy`, shouldn't this be `galaxy/openshift-applier/...` ?